bundles/ranvier-commands/commands/config.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+module.exports = (srcPath) => {
+  const Broadcast = require(srcPath + 'Broadcast');
+  const Parser = require(srcPath + 'CommandParser').CommandParser;
+  const Data = require(srcPath + 'Data');
+
+  return {
+    usage: 'config <set/list> [setting] [value]',
+    command: (state) => (args, player) => {
+      args = args.trim();
+
+      if (!args.length) {
+        Broadcast.sayAt(player, 'Configure what?');
+        return state.CommandManager.get('help').execute('config', player);
+      }
+
+      const possibleCommands = ['set', 'list'];
+      const splitArgs = args.split(' ');
+      const command = splitArgs[0];
+
+      if (possibleCommands.indexOf(command) === -1) {
```

Comment:
With ES6, this can be shortened to `if (possibleCommands.includes(command)) { ... `